css/settings/_variables.scss

```diff
@@ -6,7 +6,7 @@ $asset-path: "../";
 $font-path: "#{$asset-path}fonts";
 $image-path: "#{$asset-path}img";
 
-$external-href: '[href^="http"]:not([href*="designsystem.digital.gov"])';
+$external-href: '[href^="http"]:not([href*="designsystem.digital.gov"]):not([href*="touchpoints.app.cloud.gov/touchpoints/5c2410e4/submit"])';
```

Comment:
I'm wondering if there might be a more universal fix for this. It might make sense to instead apply `display: inline-block` (the display applied to all `usa-button` elements in USWDS) to all external links with `usa-button` styling. Something like this:

```
.usa-button#{$external-href} {
  display: inline-block;
}
```
 This would let us maintain the fix even if we change survey urls and could help prevent issues with other external links styled as buttons.

Reply:
Love this idea! Great thinking. This will definitely make it more future proof. Also let me play with sass interpolation a bit more 🤩 

Resolved in [7dde406](https://github.com/uswds/uswds-site/pull/2221/commits/7dde406cafdceb6bcb0185f44e3843570d56b246)